vm/ubpf_jit_arm64.c

```diff
@@ -1367,6 +1384,25 @@ resolve_leas(struct jit_state* state)
     return true;
 }
 
+static bool
+resolve_local_calls(struct jit_state* state)
+{
+    for (unsigned i = 0; i < state->num_local_calls; ++i) {
+        struct patchable_relative local_call = state->local_calls[i];
+
+        int32_t target_loc;
+        assert(local_call.target_offset == 0);
+        assert(local_call.target_pc != TARGET_PC_EXIT);
+        assert(local_call.target_pc != TARGET_PC_RETPOLINE);
+        target_loc = state->pc_locs[local_call.target_pc];
```

Comment:
Just curious: Is there a reason that you have the declaration separate from the initialization?